mollie/api/objects/order.py

```diff
@@ -202,3 +203,7 @@ def get_shipment(self, resource_id):
     def update_shipment(self, resource_id, data):
         """Update the tracking information of a shipment."""
         return Shipments(self.client).on(self).update(resource_id, data)
+
+    def create_payment(self, data=None):
```

Comment:
I don't there's any reason to allow data to be None, as you can't create a valid payment without data.